### PR TITLE
Test on Node.js 8

### PR DESCRIPTION
**Pull request by nicolo-ribaudo**
https://github.com/babel/babel-loader/blob/a4d17c0a6a0693bf970a65dd0a1cca783f1c03a6/package.json#L10

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -48,3 +48,32 @@ jobs:
...
+        env:
+          YARN_NODE_LINKER: node-modules
+      - name: Install webpack 4
+        run: yarn add -D webpack@4
+        env:
+          YARN_NODE_LINKER: node-modules
+      - name: Build babel-loader
+        run: yarn run build
+        env:
+          YARN_NODE_LINKER: node-modules
```
Comment (JLHwung) [inline .github/workflows/ci.yml:71]: The env `YARN_NODE_LINKER ` can be moved to the job scope: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#env

File: package.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
   ],
   "main": "lib/index.js",
   "engines": {
-    "node": ">= 6.9"
+    "node": ">= 8.9"
```
Comment (nicolo-ribaudo) [inline package.json:10]: This was a breaking change in https://github.com/babel/babel-loader/pull/822, but no one complained :shrug: 

Also, AVA doesn't work on Node.js 8 so I'm running a smaller single test.

Comment (existentialism) [reply package.json:10]: 🤔

Comment (nicolo-ribaudo) [reply package.json:10]: `schema-utils`, introduced in that PR, uses some syntax that is not supported on Node.js 6.

**Review by existentialism — APPROVED** [review]

**Pull request merged**